common/archiver/README.md

```diff
@@ -0,0 +1,69 @@
+## What
+
+This README explains how to create new implementations for the History and Visibility Archiver Interface.
+
+## Steps
+
+1. Create a new directory in the `archiver` folder. The structure should looks like the following:
+
+```
+./common/archiver
+  - filestore/                  -- Filestore implementation 
+  - provider/
+      - provider.go             -- Provider of archiver instances
+  - yourImplementation/
+      - historyArchiver.go      -- HistoryArchiver implementation
```

Comment:
Add files for historyArchiver_test.go and visibilityArchiver_test.go

Reply:
Done.